pom.xml

```diff
@@ -52,6 +52,7 @@
         <spring.osgi.core.version>1.2.1</spring.osgi.core.version>
 
         <javax.persistence.version>2.1.0</javax.persistence.version>
+        <exam.version>4.2.0</exam.version>
```

Comment:
Thanks for pointing that out. We have removed it now. The changes will be visible in next pull request.